common/theme.c

```diff
@@ -108,13 +118,15 @@ void themeStartup(ThemePreset preset) {
                 .backgroundColor = background,
                 .highlightColor = highlight,
                 .separatorColor = separator,
+                .borderColor = borderColor,
+                .borderTextColor = borderTextColor,
                 .enableWaveBlending = waveBlending,
-                .buttonAText = AText,
-                .buttonBText = BText,
                 //.buttonAImage = button_a_dark_bin,
```

Comment:
Remove these 3 lines.

Reply:
what three lines?

Reply:
wouldnt i want to say .hbmenuLogoimage = themeDefault->hbmenuLogoImage ?

Reply:
i can remove the comments though... which is what i am assuming you want me to remove

Reply:
i removed the comments... and changed the last line to set it to the themeDefault->hbmenuLogoImage